Makefile

```diff
@@ -0,0 +1,74 @@
+NAME=pixelizer
+
+SHELL := env DOCKER_REPO=$(DOCKER_REPO) $(SHELL)
+DOCKER_REPO?=eu.gcr.io/melsoft-infra
```

Comment:
I don't know much about dockers and makefiles, but doesn't this mean we won't be able to compile if this docker repo is down someday ? Also, it seems this repo has the name of your company. I don't know if it is relevant to use your company's docker repo here.

Reply:
Ooops, sorry -that was a copy-paste issue 😅

Reply:
If you mark this as resolved please provide an answer about it so I can know if you will fix it